test/regress/regress1/nl/issue3441.smt2

```diff
@@ -0,0 +1,8 @@
+(set-logic QF_NIA)   
```

Comment:
I am not sure, but did you miss the headers 
```
; COMMAND-LINE
; EXPECT
```

Reply:
This isn't necessary if there aren't any specific options you are using or output you are expecting (beyond a status).